CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+v1.8.0/ 2020-03-14
+===================
+
+**No changes since v1.8.0-RC2**
```

Comment:
Include the changes done in 1.8.0 and all of its RC builds.

Reply:
Done